lib/install.js

```diff
@@ -120,6 +120,11 @@ function handleResolvedPkg (dir, pkg, cb) {
       // package level.
       fs.rename.bind(null, tmpDest, finalDest)
     ], function (err) {
+      // Include package.json's metadata.
+      // `pkg` at this point just comes from the resolver, which only has
+      // { name, version, uid, shasum, tarball }.
+      pkg = assign({}, require(path.join(dir, pkg.uid, 'package.json')), pkg)
```

Comment:
Sorry, maybe I missed something. 

That's why I was going to fix it at weekend, but not in between my main job duty :smile: